tests/contract/test_resource_client.py

```diff
@@ -1373,6 +1373,30 @@ def test_compare_should_throw_exception(resource_client):
         logging.debug("This test expects Assertion Exception to be thrown")
 
 
+@pytest.mark.parametrize(
+    "inputs,outputs,schema_fragment",
+    [
+        (
+            {"CollectionToCompare": ["item1", "item2", "item3"]},
+            {"CollectionToCompare": ["item3", "item2", "item1"]},
+            {"properties": {"CollectionToCompare": {"insetionOrder": False}}},
```

Comment:
Typo in insertion. How did the tests work correctly with this Typo!

Reply:
`try-catch` actually caught the exception...